deploy/base/daemonset/kustomization.yaml

```diff
@@ -13,7 +13,6 @@ configMapGenerator:
     behavior: create
     literals:
       - SWISNAP_ENABLE_APACHE=false
-      - SWISNAP_ENABLE_DOCKER=true
```

Comment:
`- SWISNAP_ENABLE_DOCKER=false`

up to you though

Reply:
The default behavior is to use CRI instead od DOCKER. Docker is being removed from base - and proper setting is in the overlay.